en/advanced_config/README.md

```diff
@@ -1,10 +1,32 @@
 # Advanced Configuration
 
-This section contains advanced configuration topics, covering vehicle-specific tuning, and less-common sensors and peripherals.
+This section contains configuration topics that are relevant when bringing up a new airframe (or significantly modifying a supported frame), and/or for OEM manufacturers.
+It covers vehicle-specific tuning, factory/OEM calibration, and more advanced/less-common configuration.
+
+Note that the [peripheral hardware](../peripherals/README.md) documentation contains configuration that is specific to particular sensors and actuators.
  
 :::tip
-The topics are useful if you are creating a new airframe type or significantly modifying a supported frame.
-Generally if you're using a [supported airframe](../airframes/airframe_reference.md#copter) (e.g. from [QGroundControl > Airframe](../config/airframe.md)) the default tuning and configuration should be acceptable. 
+If working with an existing kit or ready-to-fly airframe you should only need the [Basic Configuration](../config/README.md). 
 :::
 
-For the standard configuration appropriate to every vehicle see: [Basic Configuration](../config/README.md).
+Topics include:

```

Comment:
As an aside, I'm open to suggestions on how to better integrate the configuration information. At the moment we have 
- Basic Config 
  - the stuff everyone building an airframe will need to do - even if building a standard airframe reference frame.
  - Some of this would even need to be done for RTF or kit vehicle - such as compass calibration, which might drift
- Advanced config - vehicle specific stuff, factory imu configuration, serial port/mavlink config
- Peripherals - individual sensors and busses (e.g. UAVCAN) have some setup info. 

This structure is pretty good if you're unfamiliar with drones and you're starting with a canned multicopter airframe or a kit frame - because you only see what you need to understand the FC assembly and wiring. But if you come in with interest in VTOL or FC, or if you come in as an OEM some of this stuff is more hidden.

I am wondering if we might be better, at least for config, having

- Configuration
  - Basic Configuration
  - MC Config
  - VTOL
  - FW
  - Factory?
  - Other categories?
  
My motivation is to reduce the sidebar and keep all configuration and calibration together. I'd still keep specific sensor/actuator configuration and setup in the respective hardware docs.

Is this better/worse? Are there any other topics you'd group in "factory/OEM"? Should we consider doing it more like ArduPilot and dividing up by vehicle type (harder!)

Reply:
I don't think my opinion carries a lot of weight, but I would personally not split these up by airframe types. I think the chances of having to duplicate some information is pretty high.

I'd prefer not to stall this PR for such a restructuring discussion though, unless you think those are tightly coupled?